### DISCUSSION TITLE
Hide Tumblr Live?

### DISCUSSION THREAD

**Discussion by ghost** [AprilSylph/XKit-Rewritten#998] [Suggestion Box]
Now that Live is on desktop, this is very much needed. I am begging on hands and knees.

**Comment by marcustyphoon** [comment]
Already done; waiting on the release!

**Reply by AprilSylph** [reply]
The release is already out on Chrome, surprised that Mozilla is taking so long...

**Reply by besina** [reply]
I tried to do this with hide element but I'm out of my depths, as it'll only block one picture rather than the whole setup. I can't wait til this is implemented on Firefox! Thanks so much for working on it! <3

**Reply by marcustyphoon** [reply]
@besina This tweak is already released!

**Comment by arcanetrivia** [comment]
C'monnnn Mozilla, this just appeared for me on web and I hates it, precious.

**Reply by marcustyphoon** [reply]
Just snooze it in your Tumblr settings!

**Reply by arcanetrivia** [reply]
Didn't notice a setting had been added (I knew it existed in the app, didn't think to look for it on web). You know what annnoys me, is that it was on by default - i.e. supposedly in "snooze" mode (which is a "negative" setting... why can't it be a straightforward on/off?) - so to get the desired effect I had to turn it off and then on again. I keep having to do this in the app, too. Sigh. (If you can't tell, I'm in the camp of Tumblr users who don't want Tumblr to have a Tiktok-alike feature.)

**Reply by marcustyphoon** [reply]
That means you had snoozed it previously. But yeah, it's pretty silly that the toggle doesn't switch back.
